com.rabbitcompany.passky.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=6.0.2
```

Comment:
```suggestion
```

Version isn't what you think it is:
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html